src/Dafda.Tests/Configuration/TestConsumerServiceCollectionExtensions.cs

```diff
@@ -16,6 +16,7 @@
 
 namespace Dafda.Tests.Configuration
 {
+    [Collection("Serializing")]

```

Comment:
Seems like a good solution to the issue we saw with side effects from some unit tests 👍